client/gutenberg/extensions/publicize/editor.scss

```diff
@@ -24,12 +24,8 @@
 }
 
 .jetpack-publicize-gutenberg-social-icon {
-	font-size: 2em;
+	fill: #555d66;
```

Comment:
Should we use a Gutenberg color variable here to make migrating later on easier?

https://github.com/WordPress/gutenberg/blob/0b30c73a19f6ccb73e8d8e4a43819fc1075080b9/assets/stylesheets/_colors.scss#L11